appvalidator/testcases/locales.py

```diff
@@ -101,3 +100,17 @@ def inspect_value(value):
                              "information listed in this locale will not be "
                              "stored or displayed to users.",
                              "Unsupported locale: %s" % locale])
+
+    if not "languages-provided" in manifest:
+        return
```

Comment:
I'm not crazy about this return. It seems like it will be easy to not notice and add code after this and result in not fun.

Can you either flip the `if` and nest the code or remove the `if` and change the languages line to:

```python
languages = set(manifest.get("languages-provided", {}).keys())
```